include/exiv2/asfvideo.hpp

```diff
@@ -1,7 +1,9 @@
 // SPDX-License-Identifier: GPL-2.0-or-later
 // Spec : Advanced Systems Format (ASF) Specification : Revision 01.20.05 :
 // https://exse.eyewated.com/fls/54b3ed95bbfb1a92.pdf
-#pragma once
```

Comment:
Hm, and I thought that `#pragma once` is the preferred include guard.

Reply:
Nope.

clang-tidy did this change itself.

Reply:
This commit replaces all of them.